packages/medusa/src/loaders/helpers/routing/index.ts

```diff
@@ -665,7 +665,7 @@ export class RoutesLoader {
         /**
          * Require the admin to be authenticated
          */
-        this.router.use(descriptor.route, authenticate())
+        this.router.use(descriptor.route, authenticateLegacy())
```

Comment:
question: Is this correct? I would expect no V1-related code to still be relevant in the core

Reply:
It is temporary and will be migrated later, this pr was to fix the authenticate middleware not being exported to unblock shahed.

Reply:
Got it!